models.py

```diff
@@ -14,6 +14,28 @@ def deactivate(self):
         self.active = False
         self.deactivated_at = datetime.datetime.utcnow()
 
+
+class GenericMixin:
+    @classmethod
+    def create(cls, **kwargs):
+        instance = cls(**kwargs)
+        return instance.save()
+
+    def update(self, **kwargs):
+        for attr, value in kwargs.items():
+            setattr(self, attr, value)
+        return self.save()
+
+    def save(self):
+        db.session.add(self)
+        try:
+            db.session.commit()
+        except Exception:
+            db.session.rollback()
+            raise Exception
+        return self
```

Comment:
I made this mixin which will allow us to write future POST/PUT endpoints with a little less code, especially when a model has a lot of fields!

I know we created `services.py` the other day, so I can rewrite this logic if we'd prefer to keep everything there.

Reply:
On second thought, this could get a little complicated since we have restrictions on the `active` and `deactivated_at` columns. I'll re-implement in `services.py` :)